docs/web/supported-browsers-and-frameworks.mdx

```diff
@@ -21,6 +21,90 @@ However, there are some gotchas to be aware of as well as common errors that you
 your stack.  The issues vary depending on the tooling in your project.  This page explains some helpful workarounds
 with links to examples.
 
+## Fetch API `keepalive`
```

Comment:
Does this make more sense being in the [Connect FAQ](https://connectrpc.com/docs/faq#typescript-and-javascript) section?